### PR TITLE
Nuget Package Icon & Symbols

### PR DESCRIPTION
**Pull request by cschuchardt88**
**Change Log**
- Added package icon to nuget package
- Added `snupkg` symbols to nuget package

_NOTE: `nuget` and `myget` require `snupkg` symbol format now._

### Icon
image

Also you want to change your workflow. Because you are packaging up `neo-cli`, `neo-gui` and others that don't need packaging for `myget` and `nuget`; you publish the files as well.

### CODE REVIEW

**Comment by shargon** [comment]
Yes, could you remove cli and gui in nuget package?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -49,8 +49,32 @@ jobs:
...
+        --configuration Debug \
+        --output ./out \
+        --version-suffix ${{ env.VERSION }}
+    - name : Pack (Neo.VM)
+      run: |
+        dotnet pack ./src/Neo.VM \
+        --configuration Debug \
+        --output ./out \
+        --version-suffix ${{ env.VERSION }}
+    - name : Pack (Neo)
```
Comment (shargon) [inline .github/workflows/main.yml:72]:
```suggestion
    - name : Pack (Neo.ConsoleService)
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -86,10 +110,33 @@ jobs:
...
+        dotnet pack ./src/Neo.Json \
+        --configuration Release \
+        --output ./out
+    - name : Pack (Neo.VM)
+      if: steps.check_tag.outputs.statusCode == '404'
+      run: |
+        dotnet pack ./src/Neo.VM \
+        --configuration Release \
+        --output ./out
+    - name : Pack (Neo)
```
Comment (shargon) [inline .github/workflows/main.yml:131]:
```suggestion
    - name : Pack (Neo.ConsoleService)
```

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -49,8 +49,32 @@ jobs:
...
       with:
         dotnet-version: ${{ env.DOTNET_VERSION }}
-    - name: Pack with dotnet
-      run: git rev-list --count HEAD |xargs printf "CI%05d" |xargs dotnet pack -c Debug -o out --include-source --version-suffix
+    - name: Set Version
+      run: git rev-list --count HEAD | xargs printf 'CI%05d' | xargs -I{} echo 'VERSION={}' >> $GITHUB_ENV
+    - name : Pack (Neo)
+      run: |
+        dotnet pack ./src/Neo \
+        --configuration Debug \
```
Comment (shargon) [inline .github/workflows/main.yml:57]: --include-source ?

Comment (cschuchardt88) [reply .github/workflows/main.yml:57]: what does that do? Symbols? if so they get automatically generated on published.

Comment (shargon) [reply .github/workflows/main.yml:57]:
```
--include-symbols - it creates the symbols package.
--include-source - it creates the symbols package with a src folder inside containing the source files.
```

Comment (cschuchardt88) [reply .github/workflows/main.yml:57]: Oh ya i we can do that.

Comment (cschuchardt88) [reply .github/workflows/main.yml:57]: that won't work see https://github.com/NuGet/Home/issues/8589, spec doesnt allow it anymore, cause of the new format.

**Review by shargon — APPROVED** [review]

**Comment by shargon** [comment]
@cschuchardt88 error : 'CI01440' is not a valid version string. (Parameter 'value')

**Comment by cschuchardt88** [comment]
ok let me look

**Comment by cschuchardt88** [comment]
its doing same as before `git rev-list --count HEAD |xargs printf "CI%05d" |xargs dotnet pack -c Debug -o out --include-source --version-suffix`

**Comment by cschuchardt88** [comment]
@shargon 
image

**Comment by cschuchardt88** [comment]
Oh you know what it is, the `Directory.Build.props` isn't being used. I think its a bug with, cause it should be using parent directory. Ill fix, had problem with this on `neo-express`

**Pull request merged**